### PR TITLE
Bug 2009131: kubevirt: add more test about vmi

### PR DESCRIPTION
**Pull request by gouyang**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/vm/vmi.spec.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    it('ID(CNV-4041) VMI disk tab', () => {
+      tab.navigateToDisk();
+      // TODO: enable below test once the 'Add Disk' is removed
+      // cy.contains(diskDialog.addDisk).should('not.exist');
+      cy.byLegacyTestID('kebab-button').should('be.disabled');
+    });
+  });
+
+  describe('Test fitler on VMs list view with VMI', () => {
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/vm/vmi.spec.ts:60]: nitpick: "fitler" => "filter"

**Comment by gouyang** [comment]
@pcbailey @yaacov Request review again, notes for review:
1. I noticed that download URL image from github is failed in a couple of PR's test results, so make them downstream only.
2. Revert the changes in `selectProject` as the flake still exists, I changed the tests `action/clone.ts' instead.
3. Try to fix flake in `source-image` test here.

**Pull request merged**